### PR TITLE
Create Leap_year.py

### PR DESCRIPTION
**Pull request by rishikant2000**
Check if year is a leap year or not

### CODE REVIEW

File: Leap_year..py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Python program to check if year is a leap year or not
+
+year = 2000
+
```
Comment (tausiq2003) [inline Leap_year..py:4]: Here there shoule be a user input but its 2000

Comment (tausiq2003) [reply Leap_year..py:4]: change it

Comment (tausiq2003) [reply Leap_year..py:4]: Please add user input

**Comment by tausiq2003** [comment]
Hey there add user input.
Reply whether I will close it or not.

File: Leap_year..py (hunk 1 above)
Comment (rishikant2000) [reply Leap_year..py:4]: Wait I am doing it.

**Pull request merged**